network/network.go

```diff
@@ -54,76 +57,90 @@ func New(ctx context.Context, opts ...NetworkCustomizer) (*testcontainers.Docker
 
 // NetworkCustomizer is an interface that can be used to configure the network create request.
 type NetworkCustomizer interface {
-	Customize(req *types.NetworkCreate)
+	Customize(req *types.NetworkCreate) error
```

Comment:
Personally for a project like this I would add it while we have the chance, otherwise you need to do a breaking version change down the line if we find a case which does need to report an error.

I'm sure the container customize method was thought not to need an error return at one point too.

Reply:
My experience is that the network API is not very much used :/ but with your explanation, I'm fine adding it